articles/cognitive-services/Computer-vision/computer-vision-resource-container-config.md

```diff
@@ -128,7 +128,7 @@ The following Docker examples are for the recognize text container.
 
 ### Logging example with environment variable
 
-  ```Docker
+  ```
   SET Logging:Console:LogLevel=Information
```

Comment:
@tfosmark Inside this block there is a `SET`, so you might be able to guess that it is being run from a Windows command prompt vs. a bash shell, but most examples that were remove where for single line calls to the `docker` executable